openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/task-tab.less

```diff
@@ -19,15 +19,17 @@
   }
 }
 
-.task-feed-message {
-  height: 100%;
-  display: flex;
-  align-items: center;
-  flex-wrap: wrap;
+.task-feed-message-container {
+  .task-feed-message {
+    height: 100%;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
 
-  &:hover {
-    .ant-typography {
-      color: @primary-color;
+    &:hover {
+      .ant-typography {
+        color: @primary-color;
+      }
```

Comment:
this style seems to be repeating, can you check?

Reply:
@ShaileshParmar11 so, we have this style but both of them are not independent, they are used with the hierarchy of their parent.

Reply:
@ShaileshParmar11  make the CSS resuable from a single location!